src/AstGen.zig

```diff
@@ -2355,6 +2355,7 @@ fn varDecl(
                 .name = ident_name,
                 .ptr = init_scope.rl_ptr,
                 .token_src = name_token,
+                .is_comptime = true,
```

Comment:
Is this correct?

Reply:
I put this as true since we have no way to know in astgen if a const is passable to lower scopes or not, so we pass all of it off to sema (with decl val ofc).

Reply:
That seems like the best approach.